icons/forgejo/forgejo-plain.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><g fill="#f9f9f9" stroke="#000"><path stroke-width="15.166" d="M37.261 105.534v-59.45c0-16.754 13.581-30.33 30.33-30.33h12.133"/><path stroke-width="15.166" d="M37.261 105.534V87.332c0-16.75 13.581-30.33 30.33-30.33h12.133"/><path stroke-width="9.099" d="M99.136 15.754c0 6.028-4.886 10.918-10.918 10.918-6.029 0-10.918-4.89-10.918-10.918 0-6.033 4.89-10.922 10.918-10.922 6.032 0 10.918 4.89 10.918 10.922zm0 41.248c0 6.033-4.886 10.918-10.918 10.918-6.029 0-10.918-4.885-10.918-10.918 0-6.028 4.89-10.918 10.918-10.918 6.032 0 10.918 4.89 10.918 10.918zM48.18 112.81c0 6.033-4.89 10.922-10.919 10.922-6.032 0-10.918-4.89-10.918-10.922 0-6.028 4.886-10.918 10.918-10.918 6.029 0 10.918 4.89 10.918 10.918zm0 0"/></g></svg>
```

Comment:
It looks like there are two white line left from editing.
![image](https://github.com/devicons/devicon/assets/482423/4ba96e35-505a-44c5-89fc-9105d844c54a)